sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -105,6 +105,18 @@ class BlobClient(StorageAccountHostsMixin):  # pylint: disable=too-many-public-m
         account URL already has a SAS token. The value can be a SAS token string, and account
         shared access key, or an instance of a TokenCredentials class from azure.identity.
         If the URL already has a SAS token, specifying an explicit credential will take priority.
+    :keyword int max_block_size: The maximum upload size used for a block blob. If the
+        block blob exceeds this size, it will be uploaded in chunks. Defaults to 4*1024*1024.
```

Comment:
max_block_size: the maximum chunk size for uploading a block blob in chunks. Defaults to 4*1024*1024.